WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -57,6 +74,16 @@ class PostListMainViewModel @Inject constructor(
 
     fun start(site: SiteModel) {
         this.site = site
+
+        val authorFilterSelection: AuthorFilterSelection = prefs.postListAuthorSelection
+
+        _updatePostsPager.value = authorFilterSelection
+        _viewState.value = PostListMainViewState.Idle(
+                isFabVisible = FAB_VISIBLE_POST_LIST_PAGES.contains(POST_LIST_PAGES.first()),
```

Comment:
We don't have a current value when start is called, so we are just grabbing the first tab since we don't current store in prefs which tab they were last on

Reply:
The problem is that this code may not do exactly what we want if we decide to store/restore the current page. Having said that, I think this one ties to [this comment](https://github.com/wordpress-mobile/WordPress-Android/pull/9484#discussion_r269345970) a bit, so let's leave it for now because I am not entirely sure what's the best way to handle it.

Reply:
If we were to store/restore the current page that code would be put right here and replace the code highlighted by this comment. Which was my intention here, provide the current default which could be easily replaced.

Reply:
I don't agree with it completely but it sounds reasonable, let's leave this as is unless other changes require a change here. Thanks.